src/App.tsx

```diff
@@ -15,6 +15,12 @@ const HeaderDiv = styled.div`
   align-items: center;
   justify-content: space-between;
   padding: 0px 40px;
+
+  .left-name{
+    @media (max-width: ${props => props.theme.breakpoints[0]}) {
```

Comment:
Use this principle https://www.styled-components.com/docs/advanced#media-templates

Reply:
Everywhere you want to use medias

Reply:
you mean i should use `@media (max-width: 700px)`,rather than this structure `@media (max-width: ${props => props.theme.breakpoints[0]})`?

Reply:
I meant to use media framework like this one:
 ${media.desktop`background: dodgerblue;`}